srml/balances/src/mock.rs

```diff
@@ -47,6 +47,7 @@ impl system::Trait for Runtime {
 }
 impl Trait for Runtime {
 	type Balance = u64;
+	type WeightToFee = ();
```

Comment:
You might want to change this to make the tests in balances pass. If they are tests that check make_payment, they should be work now with the new config type as well (as a good demonstration that the whole thing is flexible).

Reply:
tests are passing, but that doesn't mean the logic is 100% sound; I'll keep this unresolved